docs/jobs/state.md

```diff
@@ -148,19 +148,32 @@ The input state looks like this:
 
 ### Cron triggered runs
 
-On the platform, when a Run is triggered by a cron job, the input state will the
-final output state of the **last succesful run** for this workflow. This allows
-each subsequent run to _know_ about previous runs—i.e., you can pass information
-from one run to another even if they happen days apart.
+#### What are cron jobs?
+
+A **cron job** is a time-based scheduler that automatically runs your workflow at specified intervals. Think of it like setting an alarm clock for your workflow. It will run automatically at the times you configure, whether that's every hour, daily at 9 AM, or on the first day of each month.
+
+
+#### How cron input state works
+
+When a run is triggered by a cron job, the input state will be the final output state of the **last successful run** of the first step in the workflow. This allows each subsequent run to _know_ about previous runs. In other words, you can pass information from one run to another even if they happen days apart.
 
 ```js
 {
-  ...finalStateOfLastSuccessfulRun,
+  ...finalStateOfLastSuccessfulRunOfStep1,
 }
 ```
 
-If this is the first time the workflow has run, the initial state will simply by
-an empty Javascript object: `{}`
+**Example scenario**: If your workflow fetches new records and the last run processed records up to ID 1000, the next cron run will start with that information and can fetch records starting from ID 1001.
+
+If this is the first time the workflow has run, the initial state will simply be an empty JavaScript object: `{}`
+
+#### Overriding cron input
+
+You can always **manually run** a cron-triggered workflow with:
+- **Empty input**: `{}` - starts fresh without previous state.
+    - If your manual run succeeds, the next scheduled cron run will start with whatever output state your manual run produced.
```

Comment:
Good. This is super important. I would remove the bullet point, I think its clearer that way